Shared/mods/deathmatch/logic/CDebugHookManager.cpp

```diff
@@ -132,7 +136,7 @@ bool CDebugHookManager::RemoveDebugHook( EDebugHookType hookType, const CLuaFunc
 ///////////////////////////////////////////////////////////////
 void CDebugHookManager::OnLuaMainDestroy( CLuaMain* pLuaMain )
 {
-    for( uint hookType = EDebugHook::PRE_EVENT ; hookType <= EDebugHook::POST_FUNCTION ; hookType++ )
+    for( uint hookType = EDebugHook::PRE_EVENT ; hookType <= EDebugHook::POST_EVENT_FUNCTION ; hookType++ )

```

Comment:
Could `DUMMY_LAST_DEBUG_HOOK` be used here, and defined where the enums are defined?